python/pyspark/ml/feature.py

```diff
@@ -53,6 +53,18 @@ class Binarizer(JavaTransformer, HasInputCol, HasOutputCol):
     >>> params = {binarizer.threshold: -0.5, binarizer.outputCol: "vector"}
     >>> binarizer.transform(df, params).head().vector
     1.0
+     >>> import tempfile
```

Comment:
Good to know that. I'll update mine according to yours after it getting
merged.

2016年2月14日星期日，Holden Karau <notifications@github.com> 写道：

> In python/pyspark/ml/feature.py
> <https://github.com/apache/spark/pull/11203#discussion_r52852811>:
>
> > @@ -53,6 +53,18 @@ class Binarizer(JavaTransformer, HasInputCol, HasOutputCol):
> >      >>> params = {binarizer.threshold: -0.5, binarizer.outputCol: "vector"}
> >      >>> binarizer.transform(df, params).head().vector
> >      1.0
> > +     >>> import tempfile
>
> So as ended up being a follow up to #10999
> <https://github.com/apache/spark/pull/10999> we might want to simplify
> this a bit so it is more like an example since doctests are also (ideally)
> readable by users - maybe waiting for #11197
> <https://github.com/apache/spark/pull/11197> and then following the
> pattern in that PR. cc @mengxr <https://github.com/mengxr>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/11203/files#r52852811>.
>


-- 
Cheers
-----------------------------------
Xusen Yin    (尹绪森)
LinkedIn: https://cn.linkedin.com/in/xusenyin